skt/runner.py

```diff
@@ -485,6 +485,29 @@ def __jobsubmit(self, xml):
 
     def run(self, url, release, wait=False, host=None, uid="",
             arch=platform.machine(), reschedule=True):
+        """
+        Main method for testing in Beaker.
+
+        Args:
+            url:
+            release:    NVR of the tested kernel.
+            wait:       False if skt should exit after submitting the jobs,
+                        True if it should wait for them to finish.
+            host:       Force testing on a machine with provided hostname.
+            uid:        Username jobs should be submitted under. Can be
+                        retrieved by running `bkr whoami`.
+            arch:       Architecture of the machine the tests should run on.
+                        Defaults to architecture of the current machine skt is
+                        running on if not specified.
```

Comment:
It would be good to mention the architecture names are those expected by Beaker. I'm not sure if they will always match `platform.machine()`, BTW.

Reply:
From what I've seen so far they do match, but I can't find any official docs. If you have any information about that, please link them and I'll make sure things work.

Reply:
I don't have any information about that. A quick look through Beaker documentation, hasn't revealed anything of value. I think it's just good enough to mention those are what Beaker expect, and e.g. say that you need to go to Beaker UI and see what it accepts, or just say "Consult your Beaker installation for accepted architectures". Main thing is that it's not just some abstract architecture, and you can find a supported list somehow. If they do match what `platform.machine()` or anything else returns, great, but we cannot really guarantee that, and it's best to consult Beaker.